packages/enterprise-search/docs/index.asciidoc

```diff
@@ -0,0 +1,17 @@
+= enterprise-search-js
+
+:doctype:           book
+
+include::{asciidoc-dir}/../../../shared/attributes.asciidoc[]
```

Comment:
Here's what we have in the main Enterprise Search docs. This is probably the most up to date version of these includes:

```suggestion
include::{docs-root}/shared/versions/stack/{source_branch}.asciidoc[]
include::{docs-root}/shared/attributes.asciidoc[]
```